### PR TITLE
fix: gradebook table rtl tooltip position

### PR DESCRIPTION
**Pull request by vlasovmichael**
- Fixed gradebook table arrow tooltip position in RTL direction

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vlasovmichael! I've created [OSPR-6431](https://openedx.atlassian.net/browse/OSPR-6431) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@vlasovmichael Thank you for your contribution. I am lining this up for our review.

**Comment by muselesscreator** [comment]
Please provide a screenshot of the feature working in an RTL language.

**Comment by leangseu-edx** [comment]
Alternatively, you can use add flip attribute to `OverlayTrigger`. It would flip the overlay when it is out of bound.
```
 <OverlayTrigger
  trigger={['hover', 'focus']}
  key="left-basic"
  placement='left'
  flip
  overlay={(
    <Tooltip id="course-grade-tooltip">
      <FormattedMessage {...messages.totalGradePercentage} />
    </Tooltip>
  )}
>
```

**Comment by vlasovmichael** [comment]
> Please provide a screenshot of the feature working in an RTL language.

image

**Comment by vlasovmichael** [comment]
> Alternatively, you can use add flip attribute to `OverlayTrigger`. It would flip the overlay when it is out of bound.
> 
> ```
>  <OverlayTrigger
>   trigger={['hover', 'focus']}
>   key="left-basic"
>   placement='left'
>   flip
>   overlay={(
>     <Tooltip id="course-grade-tooltip">
>       <FormattedMessage {...messages.totalGradePercentage} />
>     </Tooltip>
>   )}
> >
> ```

@leangseu-edx Thanks for the advice, but it doesn't work

**Review by muselesscreator — DISMISSED** [review]
Please fix unit tests

File: src/components/GradesView/GradebookTable/LabelReplacements.test.jsx [hunk 1]
```diff
@@ -16,6 +17,32 @@ jest.mock('@edx/paragon', () => ({
   Tooltip: () => 'Tooltip',
 }));
 
+configureI18n({
```
Comment (leangseu-edx) [inline src/components/GradesView/GradebookTable/LabelReplacements.test.jsx:20]: I would suggest updating `setupTest.js`
```javascript
jest.mock('@edx/frontend-platform/i18n', () => {
  const i18n = jest.requireActual('@edx/frontend-platform/i18n');
  const PropTypes = jest.requireActual('prop-types');
  return {
    ...i18n,
    intlShape: PropTypes.shape({
      formatMessage: jest.fn(msg => msg.defaultMessage),
    }),
    defineMessages: m => m,
    FormattedMessage: () => 'FormattedMessage',
    getLocale: jest.fn(),
  };
});
```

Then you can make more verbose snapshot for rtl and ltr overlay.
```javascript
describe('snapshot', () => {
  test('right to left overlay placement', () => {
    getLocale.mockImplementation(() => 'en');
    el = shallow(<TotalGradeLabelReplacement />);
    expect(el).toMatchSnapshot();
  });
  test('left to right overlay placement', () => {
    getLocale.mockImplementation(() => 'ar');
    el = shallow(<TotalGradeLabelReplacement />);

    expect(el).toMatchSnapshot();
  })
});
```

Comment (leangseu-edx) [reply src/components/GradesView/GradebookTable/LabelReplacements.test.jsx:20]: And you won't need to define this configureI18n, which I consider inconsistent for everyone's use case.

Comment (vlasovmichael) [reply src/components/GradesView/GradebookTable/LabelReplacements.test.jsx]: @leangseu-edx Thanks a lot! Fixed

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Pull request merged**